src/llvm/pass_registry.cr

```diff
@@ -9,17 +8,32 @@ struct LLVM::PassRegistry
   def initialize(@unwrap : LibLLVM::PassRegistryRef)
   end
 
-  Inits = %w(core transform_utils scalar_opts obj_c_arc_opts vectorization inst_combine ipo instrumentation analysis ipa code_gen target)
+  {% begin %}
+    Inits = [
+      "initialize_core",
+      "initialize_transform_utils",
+      "initialize_scalar_opts",
+      {% unless LibLLVM::IS_LT_160 %} "initialize_obj_c_arc_opts", {% end %}
+      "initialize_vectorization",
+      "initialize_inst_combine",
+      "initialize_ipo",
+      {% unless LibLLVM::IS_LT_160 %} "initialize_instrumentation", {% end %}
+      "initialize_analysis",
+      "initialize_ipa",
+      "initialize_code_gen",
+      "initialize_target",
+    ]
+  {% end %}
 
   {% for name in Inits %}
-    def initialize_{{name.id}}
-      LibLLVM.initialize_{{name.id}} self
+    def {{name.id}}
+      LibLLVM.{{name.id}} self
     end
   {% end %}
 
   def initialize_all
     {% for name in Inits %}
-      initialize_{{name.id}}
+      {{name.id}}
     {% end %}
```

Comment:
Suggestion: this change could be avoided by keeping the name list prefix-less, as it was before.

Reply:
Agreed, _but_ it took me far too much time to understand why the linker needed the `LLVMInitializeObjCArgOpts` and `LLVMInitializeInstrumentation` because we never call `initialize_obj_c_arc_opts` or `initialize_instrumentation`...